app/models/booth.rb

```diff
@@ -35,13 +35,17 @@ class Booth < ActiveRecord::Base
     state :to_reject
     state :rejected
     state :canceled
+    state :confirmed
 
     event :restart do
-      transitions to: :new, from: [:withdrawn, :to_accept, :to_reject, :canceled]
+      transitions to: :new, from: [:withdrawn, :rejected, :canceled, :confirmed]
```

Comment:
If a booth is confirmed, this allows me to click 'Re-submit', which doesn't make a lot of sense, does it?